app/selfserve/module/Olcs/view/layouts/base.phtml

```diff
@@ -90,16 +90,14 @@ $isLoggedIn = $currentUser->isLoggedIn();
             <div class="govuk-grid-row">
               <div class="govuk-grid-column-two-thirds">
                 <h2 class="govuk-cookie-banner__heading govuk-heading-m">
-                  <?php echo $this->escapeHtml($this->translate('cookie-banner.tell-us')); ?>
+                  <?php echo $this->translate('cookies-on-header-vehicle-operator-licensing'); ?>
```

Comment:
This will need escaping with $this->escapeHtml(). Same for the two other examples below I'm guessing.